tests/unit/insertTestResult.unitTest.ts

```diff
@@ -1223,6 +1223,78 @@ describe("insertTestResult", () => {
         });
     });
 
+    context("when inserting a non-adr HGV with null expiry Date and null certificateNumber", () => {
```

Comment:
need TRL too - uses the other schema

Reply:
mmm indeed but it's the exact change in trl too. but understandable..I will do it

Reply:
Yep, the point of the test is to make sure that that particular change, in both files, doesn't revert (again?) so need it in both, even though it's the same functionality.